pkg/uploader/kopia/snapshot.go

```diff
@@ -436,7 +446,7 @@ func Restore(ctx context.Context, rep repo.RepositoryWriter, progress *Progress,
 	}
 
 	stat, err := restoreEntryFunc(kopiaCtx, rep, output, rootEntry, restore.Options{
-		Parallel:               runtime.NumCPU(),
```

Comment:
A quick question.
Why the uploader parallel setting is not the same for backup and restore?

Reply:
Do you mean that one setting for backup and restore?

Reply:
Yes. The backup's uploader parallel setting defaults to 1, and the restore's uploader parallel setting defaults to the CPU number.